apps/wallet-mobile/src/features/Swap/common/errors.ts

```diff
@@ -0,0 +1,19 @@
+import {WrongPassword} from '../../../yoroi-wallets/cardano/errors'
+
+export const getErrorMessage = (
+  error: unknown,
+  strings: Record<'wrongPasswordMessage' | 'error' | 'missingCollateral', string>,
+) => {
+  if (error instanceof WrongPassword) {
+    return strings.wrongPasswordMessage
+  }
+  if (error instanceof Error && error.message.includes('InsufficientCollateral')) {
```

Comment:
Is it not possible to check by instance?

Reply:
https://github.com/Emurgo/yoroi/blob/9636560565f2c67bdb918edab00fed427af60248/apps/wallet-mobile/src/yoroi-wallets/cardano/types.ts#L309